docs/current/cloud/snippets/get-started/ci/actions.yml

```diff
@@ -0,0 +1,24 @@
+name: dagger
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install deps
+        run: npm ci
+      - name: Install Dagger CLI
+        run: cd /usr/local && { curl -L https://dl.dagger.io/dagger/install.sh | sh; cd -; }
+      - name: Run Dagger pipeline
+        run: dagger run node index.mjs
+        // highlight-start
+        env:
+           DAGGER_CLOUD_TOKEN: ${{ secrets.DAGGER_TOKEN }}
+        // highlight-end
```

Comment:
```suggestion
        // highlight-end
        - name: Stop Engine
          run: docker stop -t 300 $(docker ps --filter name="dagger-engine-*" -q)
```

Reply:
Note this is only important for ephemeral engines. For long lived engines, this would be a bad thing. The important part is the `-t 300` which overrides the default 10 seconds docker gives a container to stop. This gives the engine more time to finish pushing cache layers to the cloud